RubberduckBaseProject.csproj

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <Company>Rubberduck-VBA</Company>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFramework>net46</TargetFramework>
+    <LangVersion>7.2</LangVersion>
+    <SolutionDir>$(MSBuildProjectDirectory)</SolutionDir>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <RestorePackages>true</RestorePackages>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(AssemblyVersion)' == ''">
+    <AssemblyVersion>2.2.*</AssemblyVersion>
+    <FileVersion Condition=" '$(FileVersion)' == '' ">2.2.*</FileVersion>
+    <Deterministic>False</Deterministic>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Analyzer Condition=" '$(AssemblyName)' != 'RubberduckCodeAnalysis' "
```

Comment:
The condition need to include `RubberduckTests` and `RubberduckCodeAnalysisTests`; makes no sense to run the analyzers there. (note: fixed in my commit that's not pushed yet)